app/assets/javascripts/spotlight/compat.js

```diff
@@ -0,0 +1,625 @@
+/**
```

Comment:
I'm not sure. Happy to, but it's our own assembly of shims for our use of phantomjs, not the full upstream distribution of polyfills.